modules/troubleshooting-disabling-autoreboot-mco-cli.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="troubleshooting-disabling-autoreboot-mco-cli_{context}"]
+= Disabling Machine Config Operator from automatically rebooting by using the CLI
+
+To avoid unwanted disruptions from changes made by the Machine Config Operator (MCO), you can use the CLI to modify the machine config pool (MCP) to prevent automatic rebooting after the Operator makes changes to the machine config.
```

Comment:
Although maybe outside of the scope of this PR, I think we should generally move away from this phrasing `to prevent automatic rebooting after the Operator makes changes to the machine config`.

It is phrased such that the reboot is prevented, whereas in reality, pausing will stop **all updates** from being applied, which means that your cluster is at risk of not receiving important updates, or rotating certificates correctly, which would eventually break your cluster if paused too long.

We can maybe say `to stop all updates to your RHCOS nodes, including OS, security, certificate, and any other machineconfig related updates, thereby preventing the MCO from performing a reboot when you do not want your workloads to be disrupted. This should not be done for long periods of time`